### DISCUSSION TITLE
Welcome to Midden Discussions!

### DISCUSSION THREAD

**Discussion by bryanrcarlson** [CafIncubator/Midden#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about your interest in Midden.

**Comment by riemino** [comment]
Hey Bryan, looks like your invitation to contribute to this project expired. Could you please resend?  Gracias!

**Reply by bryanrcarlson** [reply]
Hey @riemino ! I see you were able to create a PR; is this what you wanted contributor status for? 

Let me know if you still want to be a contributor and I'll resend. (Part of me wants to keep you as a non-member though so we can better test/learn about public contributions.)

**Reply by riemino** [reply]
No, I'm ok staying as I am.  I just though you wanted me to be a contributor since I saw a message but the invite had expired.
